java-spiffe-core/src/test/java/io/spiffe/svid/jwtsvid/JwtSvidParseAndValidateTest.java

```diff
@@ -154,7 +154,7 @@ void testParseAndValidate_nullAudience_throwsNullPointerException() throws JwtSv
                         .jwtBundle(jwtBundle)
                         .expectedAudience(Collections.singleton("audience"))
                         .generateToken(() -> HS256TOKEN)
-                        .expectedException(new JwtSvidException("Unsupported token signature algorithm HS256"))
+                        .expectedException(new JwtSvidException("Unsupported JWT token algorithm: HS256"))
```

Comment:
nit: `JWT token` is a little redundant, I think just removing `token` from the message should be fine